redi-connect-front/src/pages/app/profile/mentor/ProfileMentor.tsx

```diff
@@ -1,100 +1,53 @@
-      margin: '3px',
-      height: '20px'
-    },
-    personalDescription: {
-      marginTop: theme.spacing(6),
-      marginBottom: theme.spacing(6),
-      overflowWrap: 'break-word'
-    }
-  })
-
-export const ProfileMentor = withStyles(styles)(
-  ({ mentor, classes }: Props) => {
-    const profile = getRedProfile()
+export const ProfileMentor = ({ mentor }: Props) => {
+  return (<>
+    <Columns vCentered breakpoint="mobile">
+      <Columns.Column size={3}>
+        <Avatar profile={mentor} />
+      </Columns.Column>
+      <Columns.Column size={9}>
+        <Heading>{`${mentor.firstName} ${mentor.lastName}`}</Heading>
+      </Columns.Column>
+    </Columns>
 
-    return <>
-      <Grid container spacing={2}>
-        <Grid item xs={12} sm={5}>
-          {/* <Avatar
-            className={classes.avatar}
-            s3Key={mentor.profileAvatarImageS3Key}
-            style={{ width: '100%', height: '20vh' }}
-          /> */}
-        </Grid>
-        <Grid item xs={12} sm={7}>
-          <h3 style={{ fontWeight: 700, fontFamily: 'Roboto' }}>
-            <ProfileName name={`${mentor.firstName} ${mentor.lastName}`} />
-            <ProfileOccupation occupation={mentor.mentor_occupation} />
-            <ProfileWorkPlace workPlace={mentor.mentor_workPlace} />
-            <ProfileLanguages languages={mentor.languages} />
-          </h3>
-        </Grid>
-      </Grid>
+    <Element className="block-separator">
+      <ReadAbout.Some profile={mentor} />
+    </Element>
 
-      <ProfileAvailableMenteeSlots
-        totalCapacity={mentor.menteeCountCapacity}
-        currentFreeCapacity={mentor.currentFreeMenteeSpots}
-      />
-      <p>I can help you with:</p>
-      {mentor.categories.map(catId => (
-        <CategoryChip
-          key={catId}
-          className={classes.category}
-          categoryId={catId}
-        />
-      ))}
-      <p className={classes.personalDescription}>
-        {mentor.personalDescription}
-      </p>
-      {mentor.expectations && (
-        <>
-          <h4 style={{ marginBottom: 0 }}>Expectations to my mentee:</h4>
-          <p
-            className={classes.personalDescription}
-            style={{ marginTop: '0.3em' }}
-          >
-            {mentor.expectations}
-          </p>
-        </>
-      )}
+    {mentor.categories && <Element className="block-separator">
+      <ReadMentoring.Some profile={mentor} />
+    </Element>}
 
-      {(mentor.numberOfPendingApplicationWithCurrentUser === 0 && profile.userType === 'mentee') && (
-        <ConnectionRequestForm mentorId={mentor.id} />
-      )}
-      {mentor.numberOfPendingApplicationWithCurrentUser > 0 && (
-        <p>You have already applied to this mentor.</p>
-      )}
-    </>
-  }
-)
+    {(mentor.languages && (mentor.gender || mentor.age)) && <Element className="block-separator">
+      <Columns>
+        {(mentor.gender || mentor.age) && <Columns.Column>
+          <ReadPersonalDetail.Some profile={mentor} />
```

Comment:
Coming full circle back to my comment about the name `Some`: I don't see it being a useful name. It implies the user browsing this profile is at best going to see "some" personal details.

Reply:
This is a hard one, yes. To give some context to the some. (its bad when you need to explain your code, I know). I was looking to define a word for some expressing the fact that the component can select any profile or some profile. Do you have "some" better idea maybe?

Reply:
@ezonai Hmm, hard one. `Container` or `Box`, perhaps?